src/main/java/nl/tudelft/contextproject/model/level/roomIO/TileParser.java

```diff
@@ -43,13 +60,37 @@ public static void readTiles(MazeTile[][] tiles, int width, int height, int xOff
 			for (int x = 0; x < line.length; x++) {
 				int posx = x + xOffset;
 				int posy = y + yOffset;
-				if (line[x].equals("#")) {
+				if (line[x].equals(EMPTY_FORMAT)) {
 					// always overwrite old value
 					tiles[posx][posy] = null;
 				} else {
-					tiles[posx][posy] = new MazeTile(posx, posy, TileType.valueOf(line[x]));
+					tiles[posx][posy] = new MazeTile(posx, posy, format(line[x]));
 				}
 			}
 		}
 	}
+
+	/**
+	 * Translate the map file to actual type.
```

Comment:
please add a newline after this line, so the @ lines are seperated from the method description